backend/app/vending/__init__.py

```diff
@@ -285,55 +285,16 @@ def get_global_vending_config() -> VendingConfig:
     )
 
 
-@router.get("app/{appid}")
-def get_app_vending_status(appid: str) -> VendingDescriptor:
-    """
-    Retrieve the vending status for the given application.  Returns an error if
-    the appid is not known, or is not set up for vending.
-    """
-    vend = ApplicationVendingConfig.by_appid(db, appid)
-    if not vend:
-        return Response(status_code=204)
-    try:
-        shares = prices.compute_shares(appid, vend.appshare)
-    except ValueError as val_err:
-        raise VendingError(error="bad-app-share") from val_err
-
-    (
-        fee_fixed_cost,
-        fee_cost_percent,
-        fee_prefer_percent,
-    ) = prices.flathub_fee_parameters(vend.currency)
-
-    return VendingDescriptor(
-        status="ok",
-        currency=vend.currency,
-        components=shares,
-        fee_fixed_cost=fee_fixed_cost,
-        fee_cost_percent=fee_cost_percent,
-        fee_prefer_percent=fee_prefer_percent,
-    )
-
-
 @router.get("app/{appid}/setup")
 def get_app_vending_setup(appid: str, login=Depends(login_state)) -> VendingDescriptor:
     """
     Retrieve the vending status for a given application.  Returns a no
     content response if the appid has no vending setup.
-
-    If any of the currency or amount values constraints are violated
-    then a no content response will also be returned because the
-    configuration is invalid.
-
-    If you do not have the right to see the vending status for this application
-    then you will be refused.
     """
 
     if not login["state"].logged_in():
         raise VendingError(error="not-logged-in")
 
-    # TODO: Check that the calling user owns the given appid
```

Comment:
@danielsilverstone-ct Just to make sure you see that this TODO I added was incorrect (since this endpoint is used by anyone to determine if an application has vending setup)